### PR TITLE
OKTA-438747 Custom error guide multi-to-single page

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Consolidates pages of custom error guide to a single page.
- **Is this PR related to a Monolith release?** Nope.

### Resolves:
* [OKTA-438747](https://oktainc.atlassian.net/browse/OKTA-438747)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+---
+title: Customize the Okta-hosted error pages
+excerpt: Learn how to customize the Okta-hosted error pages to fit your brand.
+layout: Guides
+---
+
+This guide teaches you how to customize error pages that can appear as part of a custom domain Okta-hosted sign-in flow.
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:7]: @christinechoy-okta Could you please review the nut facts for this guide? It's my first time having a go at them. Thanks!

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+---
+
+**Learning outcomes**
+
+Customize error pages as part of the sign-in flow
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:13]:
```suggestion
Customize error pages as part of the sign-in flow.
```

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+
+**What you need**
+
+* An Okta Developer Edition organization. Don't have one? [Create one for free](/signup).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:17]:
```suggestion
* An [Okta Developer Edition organization](/signup)
```

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+* A [customized Okta URL domain](/docs/guides/custom-url-domain/), unless you are using the [Brands API](#use-the-brands-api).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:18]:
```suggestion
* A [customized Okta URL domain](/docs/guides/custom-url-domain/), unless you are using the [Brands API](#use-the-brands-api)
```

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+
+**Sample code**
+
+See our [customization examples](#customization-examples)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:22]:
```suggestion
[Customization examples](#customization-examples)
```

**Review by christinechoy-okta — COMMENTED** [review]
I reviewed the nutrition facts. Just a few nits. Thanks!

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+
+---
+
+## Overview
+
+When using an Okta-hosted flow, you can create a unique sign-in experience by providing a customized [Okta URL domain](/docs/guides/custom-url-domain/) and creating an [Style the Widget](/docs/guides/style-the-widget/style-okta-hosted/) that matches your application's look and feel.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:28]:
```suggestion
When using an Okta-hosted flow, you can create a unique sign-in experience by providing a customized [Okta URL domain](/docs/guides/custom-url-domain/) and creating a [Widget style](/docs/guides/style-the-widget/style-okta-hosted/) that matches your application's look and feel.
```

File: packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+
+```html
+<p class="technical-details">Additional Error Details: {{technicalDetails}}</p>
+```
+
+> **Note:** The following macros are only available when the Theme Builder feature is enabled in your org.
+
+### <span v-pre>`{{buttonText}}`</span>
+
+Inserts the button text based on the page context. When a user selects the button, they are directed to the `buttonHref` URL. The `{{back}}` macro is also supported for the same purpose.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md:145]:
```suggestion
Inserts the button text based on the page context. When the user selects the button, they are directed to the `buttonHref` URL. The `{{back}}` macro is also supported for the same purpose.
```

File: packages/@okta/vuepress-site/docs/release-notes/2018/index.md [hunk 8]
```diff
@@ -808,11 +808,11 @@ You can customize your Okta org by replacing the Okta domain name with a custom
 
 #### Custom Okta-hosted Sign-In Page is in Early Access
 
-You can customize the text and the look and feel of the Okta-hosted sign-in page using form controls and an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted error page](/docs/guides/custom-error-pages/overview/), this feature offers a fully customized end-user sign-in experience hosted by Okta. For details, see [Configure a custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/).
+You can customize the text and the look and feel of the Okta-hosted sign-in page using form controls and an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted error page](/docs/guides/custom-error-pages/), this feature offers a fully customized end-user sign-in experience hosted by Okta. For details, see [Configure a custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2018/index.md:811]:
```suggestion
You can customize the text and the look and feel of the Okta-hosted sign-in page by using form controls and an embedded HTML editor. When this feature is used with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted error page](/docs/guides/custom-error-pages/), it offers a fully customized end-user sign-in experience that is hosted by Okta. See [Configure a custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/) for more information.
```

File: packages/@okta/vuepress-site/docs/release-notes/2018/index.md [hunk 9]
```diff
@@ -808,11 +808,11 @@ You can customize your Okta org by replacing the Okta domain name with a custom
...
 
 #### Custom Error Page is in Early Access
 
-You can customize the text and the look and feel of error pages using an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/), this feature offers a fully customized error page. For details, see [Configure a custom error page](/docs/guides/custom-error-pages/overview/).
+You can customize the text and the look and feel of error pages using an embedded HTML editor. When used together with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/), this feature offers a fully customized error page. For details, see [Configure a custom error page](/docs/guides/custom-error-pages/).
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2018/index.md:815]:
```suggestion
You can customize the text and the look and feel of error pages by using an embedded HTML editor. When the feature is used with [custom URL domain](https://help.okta.com/okta_help.htm?id=ext_custom_url_domain) (required) and [custom Okta-hosted sign-in page](/docs/guides/style-the-widget/style-okta-hosted/), it offers a fully customized error page. See [Configure a custom error page](/docs/guides/custom-error-pages/) for more information.
```

**Review by christinechoy-okta — APPROVED** [review]
Just a few nits!

**Pull request merged**